google/resource_storage_bucket_object.go

```diff
@@ -79,6 +79,26 @@ func resourceStorageBucketObject() *schema.Resource {
 			},
 
 			"md5hash": &schema.Schema{
+				Type:     schema.TypeString,
```

Comment:
The diff is not easy to read. I basically reverted "md5hash" field back to its previous state.

And created a new field "detect_md5hash". This field is the one that will detect the changes. I improved the DiffSuppressFunc to also detect changes when the remote file changes and the `content` field is used.